pinc/faq.inc

```diff
@@ -158,7 +159,8 @@ function get_faq_url($faq, $langcode = null)
     return "";
 }
 
-function get_all_urls_for_faq($faq)
+/** @return array<string, string> */
+function get_all_urls_for_faq(string $faq): array
```

Comment:
This function appears unused now -- it was only used in `faq_central.php` -- let's delete it.

Reply:
I pushed up a commit to delete this.